server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -377,14 +377,11 @@ public void deleteScanServerFileReferences(String serverAddress, UUID scanServer
     Objects.requireNonNull(scanServerLockUUID, "Server uuid must be supplied");
     try (
         Scanner scanner = context.createScanner(DataLevel.USER.metaTable(), Authorizations.EMPTY)) {
-      scanner.setRange(ScanServerFileReferenceSection.getRange());
-      scanner.fetchColumn(new Text(serverAddress), new Text(scanServerLockUUID.toString()));
+      scanner
```

Comment:
This will be so much faster now that it does not scan the entire range anymore.

Could push the creation of the range or maybe row into ScanServerRefTabletFile.